sns_ik_lib/src/sns_position_ik.cpp

```diff
@@ -30,15 +30,41 @@ SNSPositionIK::SNSPositionIK(KDL::Chain chain, std::shared_ptr<SNSVelocityIK> ve
     m_jacobianSolver(chain),
     m_linearMaxStepSize(0.2),
     m_angularMaxStepSize(0.2),
-    m_maxIterations(100),
-    m_dt(0.2)
+    m_maxIterations(150),
```

Comment:
Now that it's running faster, I increased the max number of iterations for a few percentage points improvement in success rate.